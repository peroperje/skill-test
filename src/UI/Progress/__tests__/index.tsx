import React from 'react';
import { create } from 'react-test-renderer';

import { LinearProgress, CircularProgress } from '../index';

describe('Progress', () => {
  describe('LinearProgress', () => {
    it('Snapshot', () => {
      const component = create(<LinearProgress />);
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
  describe('CircularProgress', () => {
    it('Snapshot', () => {
      const component = create(<CircularProgress />);
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
