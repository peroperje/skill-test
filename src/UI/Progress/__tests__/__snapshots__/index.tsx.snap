// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress CircularProgress Snapshot 1`] = `
<div
  className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
  role="progressbar"
  style={
    Object {
      "height": 40,
      "width": 40,
    }
  }
>
  <svg
    className="MuiCircularProgress-svg"
    viewBox="22 22 44 44"
  >
    <circle
      className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
      cx={44}
      cy={44}
      fill="none"
      r={20.2}
      strokeWidth={3.6}
      style={Object {}}
    />
  </svg>
</div>
`;

exports[`Progress LinearProgress Snapshot 1`] = `
<div
  className="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate"
  role="progressbar"
>
  <div
    className="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate"
    style={Object {}}
  />
  <div
    className="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
    style={Object {}}
  />
</div>
`;
